data/event/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     kotlin("multiplatform")
     id("com.android.library")
+    id("com.apollographql.apollo3").version(libs.versions.apollo)
```

Comment:
Ideally this plugin information also goes into the `data:remote` module so we can have some code shared across all of the specific data layer modules.